content/docs/command-reference/remote/modify.md

```diff
@@ -406,7 +406,42 @@ application. To use a custom authentication method, use the following parameters
   ```dvc
   $ dvc remote modify myremote allow_anonymous_login true
   ```
+**Authentication examples:**
 
+Authentication example with `connection_string`:
+
+```dvc
+$ dvc remote add -d myremote azure://mycontainer/object
+$ dvc remote modify --local myremote connection_string 'storageaccountaccesskeyconnectionstring'
+$ dvc remote push
```

Comment:
Wait... We already have these examples under each config option. E.g.

![image](https://user-images.githubusercontent.com/1477535/126122611-5fbd1db0-eebe-411c-aa10-aef2109e751e.png)

The hope is that the `remote add` and `push/pull` commands are obvious to people reading this section.

Reply:
My point (and thus I included the `dvc remote push` command) is that I wanted to provide examples of correctly working combinations of authentication settings.

The already existing examples only show what parameters can be added, but not in which way they should be combined.

Reply:
> not in which way they should be combined

The ways in which they should be combined is there, pretty close to the beginning of this section:

![image](https://user-images.githubusercontent.com/1477535/127275397-aa3e3396-3726-47f4-9eee-a62bf5de2e3e.png)